pandas/tseries/resample.py

```diff
@@ -673,18 +670,15 @@ def aggregate(self, arg, *args, **kwargs):
     def _get_new_index(self):
         """ return our new index """
         ax = self.ax
-        ax_attrs = ax._get_attributes_dict()
-        ax_attrs['freq'] = self.freq
-        obj = self._selected_obj
 
         if len(ax) == 0:
-            new_index = PeriodIndex(data=[], **ax_attrs)
-            return obj.reindex(new_index)
-
-        start = ax[0].asfreq(self.freq, how=self.convention)
-        end = ax[-1].asfreq(self.freq, how='end')
+            values = []
+        else:
+            start = ax[0].asfreq(self.freq, how=self.convention)
+            end = ax[-1].asfreq(self.freq, how='end')
```

Comment:
>What exactly is the problem?

That it doesn't make conceptual sense. If we think about a `Period` as span of time with a start and an end, then what is this example doing, in concept? It's all the subperiods of the second period and only the final subperiod of the first period, but the value from the first period bleeding through to the second? 
(it makes sense with point-in-time indexes, so I can see where it came from, it's just misapplied here)

There are two operations that make sense with these interval-like items:
1. Upsample: Get all the subperiods of each period and put some transformation of the values in them
2. Downsample: Bin the values into a set of superperiods and aggregate them in some way

```python
In [67]: pd.Series(range(2), pd.period_range('1/1/1990', periods=2, freq='M')).resample('D',convention='end').pad()
Out[67]: 
1990-01-31    0
1990-02-01    0
1990-02-02    0
1990-02-03    0
1990-02-04    0
1990-02-05    0
1990-02-06    0
1990-02-07    0
1990-02-08    0
1990-02-09    0
1990-02-10    0
1990-02-11    0
1990-02-12    0
1990-02-13    0
1990-02-14    0
1990-02-15    0
1990-02-16    0
1990-02-17    0
1990-02-18    0
1990-02-19    0
1990-02-20    0
1990-02-21    0
1990-02-22    0
1990-02-23    0
1990-02-24    0
1990-02-25    0
1990-02-26    0
1990-02-27    0
1990-02-28    1
Freq: D, dtype: int64
```